### PR TITLE
Bump zio-http from 3.0.0RC09 -> 3.0.1

### PR DESCRIPTION
**Pull request by anakos**
addresses https://github.com/kaizen-solutions/trace4cats-zio-extras/issues/44

### CODE REVIEW

File: zio-http/src/main/scala/io/kaizensolutions/trace4cats/zio/extras/ziohttp/client/ZioHttpClientTracer.scala [hunk 1]
```diff
@@ -32,7 +32,7 @@ object ZioHttpClientTracer {
         span.putAll(toAttributes(request)).when(span.context.traceFlags.sampled == SampleDecision.Include)
 
       enrichWithAttributes *>
-        Client.request(requestWithTraceHeaders).tap { response =>
+        Client.streaming(requestWithTraceHeaders).tap { response =>
```
Comment (anakos) [inline zio-http/src/main/scala/io/kaizensolutions/trace4cats/zio/extras/ziohttp/client/ZioHttpClientTracer.scala:35]: request is deprecated in 3.0.0. Looking @ the zio-http source streaming seems to be aliased variant of request .

**Comment by calvinlfer** [comment]
Thank you so much 🙏🏽

**Comment by anakos** [comment]
thanks @calvinlfer , much appreciated!

**Comment by soujiro32167** [comment]
Thank you!

**Pull request merged**